internal/cmd/local/local/cmd.go

```diff
@@ -545,26 +570,44 @@ func (c *Command) handleBasicAuthSecret(ctx context.Context, user, pass string)
 		return fmt.Errorf("unable to hash basic auth password: %w", err)
 	}
 
-	data := map[string][]byte{"auth": []byte(fmt.Sprintf("%s:%s", user, hashedPass))}
-	if err := c.k8s.SecretCreateOrUpdate(ctx, corev1.SecretTypeOpaque, airbyteNamespace, "basic-auth", data); err != nil {
-		pterm.Error.Println("Could not create Basic-Auth secret")
+	secret := corev1.Secret{
```

Comment:
For a later refactor: we can add a util for instantiating a secret object as we continue to add more cases.